data_model/src/permissions.rs

```diff
@@ -52,6 +49,23 @@ pub struct PermissionToken {
     params: btree_map::BTreeMap<Name, Value>,
 }
 
+impl core::fmt::Display for PermissionToken {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(f, "{}: {{", self.name)?;
```

Comment:
if we deligate writing brackets to outer code than we could do something like this:

```rust
let comma_separated_params = format_comma_separated(self.params.iter().map(|(name, value)| format!("`{}`: `{}`", name, value)))
write!(f, "{}: {{{}}}", self.name, comma_separated_params)
```